package.json

```diff
@@ -45,7 +45,7 @@
     "tar": "2.2.1",
     "underscore": "1.8.3",
     "underscore-deep-extend": "0.0.5",
-    "yargs": "9.0.1"
+    "yargs": "12.0.1"
```

Comment:
Is there any breaking changes introduced in the major bumps here?

Reply:
"Upgrade version of yargs from 9.0.1 to 12.0.1 ( it doesn't have break changes which affect this project - it will just get bug fix and allow new features of it). The usage of the yargs is to run the commands and its usage is re-checked in this PR." (ref first commit)